src/test/java/step1/SetCollectionTest.java

```diff
@@ -26,21 +26,21 @@ void setUp() {
     }
 
     @Test
-    @DisplayName("요구사항1")
+    @DisplayName("Set의 size() 메소드를 활용해 Set의 크기를 확인")
     void setSize() {
         assertThat(numbers.size()).isEqualTo(3);
     }
 
     @ParameterizedTest
-    @DisplayName("요구사항2")
+    @DisplayName("Set의 contains() 메소드를 활용해 1, 2, 3의 값이 존재하는지를 확인")
     @ValueSource(ints = {1, 2, 3})
     void setContains(int input) {
         assertThat(numbers.contains(input)).isTrue();
     }
 
     @ParameterizedTest
     @CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false"}, delimiter = ':')
-    @DisplayName("요구사항3")
+    @DisplayName("1, 2, 3 값은 contains 메소드 실행결과 true, 4, 5 값을 넣으면 false 가 반환 확인")
```

Comment:
```suggestion
    @DisplayName("contains 메서드는 자료구조에 존재하는 원소를 전달하면 true를 반환한다.")
```
1,2,3보다는 좀 더 추상적으로 작성해도 좋습니다 ㅎ